pom.xml

```diff
@@ -92,8 +92,7 @@
     <pig.version>0.16.0</pig.version>
     <pig.classifier>h2</pig.classifier>
     <thrift-maven-plugin.version>0.10.0</thrift-maven-plugin.version>
-    <thrift.version>0.9.3</thrift.version>
-    <format.thrift.version>0.9.3</format.thrift.version>
+    <thrift.version>0.12.0</thrift.version>
```

Comment:
Please, keep the format version separated even if we bump both to 0.12.0.
(Format version is used to read/write thrift structures in parquet files and the related thrift library is therefore shaded and shipped with parquet. `thrift.version` is used for the `parquet-thrift` module where the users may use their own thrift versions.)

Reply:
@gszadovszky Sure, I've reverted the change.